src-electron/generator/helper-command.js

```diff
@@ -407,6 +407,36 @@ async function if_command_fixed_length(commandId, options) {
   return options.fn(this)
 }
 
+/**
+ * If helper that checks if a reponse command is present for a given command
+ *
+ * example:
+ * {{#if_command_has_response command 'Request'}}
+ * command has reponse
+ * {{else}}
+ * command has no reponse
+ * {{/if_command_has_response}}
+ *
+ * @param commandName Name of the command
+ * @param replaceKeyWord Check for response command after removing the string
+ * from command name
+ * @returns content based on whether the command response was found or not
+ */
+async function if_command_has_response(commandName, replaceKeyWord, options) {
```

Comment:
But this ignores commands that have a response defined via responseName, no?   As in, there are commands with responses that this function returns false for....

Reply:
That is correct. I added responseName to selectAllIncomingOrOutgoingCommandsForCluster such that helpers calling this function can have direct access to responseName which can be used in a #if helper. I have filed https://github.com/project-chip/zap/issues/402 for resolving command responses cleanly.

Reply:
I guess I don't understand how one would use this helper in practice.  Is the idea that you check for `responseName` first, and if falsy _then_ use this helper?

I guess this is all kinda broken, in that this has logic that needs to match logic in another repo, etc, etc...  Ah, well.

Reply:
Yes one needs to check responseName first and then use this.